public/index.html

```diff
@@ -45,6 +46,29 @@
                     $scope.ideas = response.data;
                 });
     });
+    app.controller('voteCtrl', function($scope, $window, $http) {
+    $scope.myFunction = function() {
+        $http({
+          method: 'POST',
+          url: '/vote',
+          headers: {
+            'Content-Type': "application/json"
+          },
+          // TODO remove hard coded urls
+          data: { "person" : "http://localhost:8080/people/1",
```

Comment:
great. It works that way too, and looks much better. Since this is going to be changed anyway as a TODO, let improve this later as you suggested in the discussion.